server/concepts/Tournament.ts

```diff
@@ -123,13 +124,53 @@ export default class Tournament implements TournamentWithoutMS {
     /* eslint-enable @typescript-eslint/no-non-null-assertion */
   }
 
-  public static async getById(tournamenId: string, token: string) {
-    const tournament = await TournamentRepository.getById(tournamenId);
+  public static async getById(tournamentId: string, token: string) {
+    const tournament = await TournamentRepository.getById(tournamentId);
     if (!tournament) return null;
     const enrichedTournament = await Tournament.enrichWithMSUsers(tournament, token);
     return enrichedTournament;
   }
 
+  public static async countPointsPerTeam(tournamentId: string) {
+    const tournament = await TournamentRepository.getById(tournamentId);
+    if (!tournament) return;
```

Comment:
```suggestion
    const tournament = await TournamentRepository.getById(tournamentId);
    if (!tournament) return null;
```
This will not change logic but We are used to returning `null` in the project. Not very important